docker/policies/yum_clean_all_missing.rego

```diff
@@ -0,0 +1,45 @@
+package appshield.DS015
+
+import data.lib.docker
+
+__rego_metadata__ := {
+	"id": "DS015",
+	"title": "Yum Clean All Missing",
+	"version": "v1.0.0",
+	"severity": "HIGH",
+	"type": "Dockerfile Security Check",
+	"description": "Need to use 'yum clean all' after using a 'yum install' command to clean package cached data and reduce image size",
+	"recommended_actions": "Add 'yum clean all' to Dockerfile",
+	"url": "https://docs.docker.com/develop/develop-images/dockerfile_best-practices/#run",
+}
+
+__rego_input__ := {
+	"combine": false,
+	"selector": [{"type": "dockerfile"}],
+}
+
+get_yum[arg] {
+	run := docker.run[_]
+	arg := run.Value[0]
+
+	regex.match("yum (-[a-zA-Z]+ *)*install", arg)
+
+	not contains_clean_after_yum(arg)
+}
+
+deny[res] {
+	args := get_yum[_]
+	res := sprintf("'yum clean all' is missed: %s", [args])
+}
+
+contains_clean_after_yum(cmd) {
+	yum_install_command := regex.find_n("yum (-[a-zA-Z]+ *)*install", cmd, -1)
+
+	install := indexof(cmd, yum_install_command[0])
```

Comment:
I think we should take the last element. It cannot detect as below though it is not practical.

```
RUN yum -y install bash && yum clean all && yum -y install zsh
```

Reply:
done